include/aspect/geometry_model/two_merged_chunks.h

```diff
@@ -0,0 +1,297 @@
+/*
+  Copyright (C) 2011 - 2015 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file doc/COPYING.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef __aspect__geometry_model_two_merged_chunks_h
+#define __aspect__geometry_model_two_merged_chunks_h
+
+#include <aspect/geometry_model/interface.h>
+#include <deal.II/grid/manifold.h>
+#include <deal.II/base/function_lib.h>
+#include <deal.II/grid/grid_out.h>
+
+namespace aspect
+{
+  namespace GeometryModel
+  {
+    using namespace dealii;
+
+    /**
+     * A geometry model class that describes a chunk of a spherical shell.
+     * In 2D, this class simulates a sector with inner and outer radius, and
+     * minimum and maximum longitude. Longitude increases anticlockwise
+     * from the positive x-axis, as per the mathematical convention of phi.
+     * In 3D, the class simulates a chunk of a sphere, bounded by arbitrary
+     * lines of longitude, latitude and radius.
+     *
+     * In addition, the lateral boundaries are split into an upper and lower
+     * part, such that different boundary conditions can be set along the
+     * same boundary. Boundary indicator names are inner and outer,
+     * lowerwest, lowereast, lowersouth, lowernorth,
+     * upperwest, uppereast, uppersouth, uppernorth.
+     * The radius at which the change is boundary indicator occurs,
+     * can be set by the user.
+     *
+     * The parameters that describe this geometry and that are read from the
+     * input file are the inner and outer radii of the shell, the minimum
+     * and maximum longitude, minimum and maximum longitude, and the
+     * number of cells initialised in each dimension.
```

Comment:
is this just the same text that was in the "chunk" geometry? If so, you may want to update it (and make the changes suggested above to the original chunk geometry, either in this pull request or a separate one).